app/models/wpcc/salary_frequency.rb

```diff
@@ -0,0 +1,27 @@
+module Wpcc
+  class SalaryFrequency
+    include ActiveModel::Model
+    attr_accessor :params_salary_frequency, :session_salary_frequency
+
+    DEFAULT_SALARY_FREQUENCIES = Hash[
+      Wpcc::YourDetailsForm::SALARY_FREQUENCIES.map do |elem|
+        [elem, elem == 'year' ? 'month' : elem]
```

Comment:
@Tr4pSt3R  I think its okay sometimes to use more advanced ruby enumerable idioms. I also dont think that 'first glance' is always achievable or necessary. Sometimes we do have to work a little harder to understand what a line of code is aiming to achieve. Please view @tomas-stefano's suggestion and lmk if you think its better and/or if you have alternative suggestions.